### DISCUSSION TITLE
Show billable hours in a different colour

### DISCUSSION THREAD

**Discussion by fzanarotti-atoms** [gwleuverink/clickup-time-tracker#34] [Ideas]
It would be useful displaying hours marked as "billable" in a different colour than "non billable" ones. Requires a Business account or greater in ClickUp to mark hours as billable.

**Comment by gwleuverink** [comment]
Thanks for the suggestion. I plan to add a billable hour toggle soon. I'll add this as well. Probably with an icon

**Reply by saragilcas** [reply]
Hi @gwleuverink 

First of all, thanks for this amazing work! :) Do you have any updates on this? I just started using this time tracker and didn't see this functionality :) 

Thanks so much!
